apis/apiextensions/v1beta1/composition_types.go

```diff
@@ -206,6 +208,7 @@ type PatchPolicy struct {
 	// +kubebuilder:validation:Enum=Optional;Required
 	// +optional
 	FromFieldPath *FromFieldPathPolicy `json:"fromFieldPath,omitempty"`
+	MergeOptions  *xpv1.MergeOptions   `json:"mergeOptions,omitempty"`
```

Comment:
@negz have we had a release with this missing?

Reply:
No, this was only merged within the last day or two.

Reply:
Nice -- thanks!